db/migrate/20240110235639_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :Photo
+      t.string :Bio
+      t.integer :PostCounter
```

Comment:
- [ ] Hello, fantastic progress! 🌟 When working with Rails, it's customary to use snake_case for column names. In the given migration, I recommend changing the column names for "Photo", "Bio" and "PostCounter" to follow this convention:

```suggestion
      t.string :photo
      t.string :bio
      t.integer :posts_counter
```